tests/com/goide/inspections/GoEmptyDeclarationInspectionTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013-2016 Sergey Ignatov, Alexander Zolotov, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.inspections;
+
+import com.goide.quickfix.GoQuickFixTestBase;
+import org.jetbrains.annotations.NotNull;
+
+public class GoEmptyDeclarationInspectionTest extends GoQuickFixTestBase {
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    myFixture.enableInspections(GoEmptyDeclarationInspection.class);
+  }
+
+  @Override
+  protected boolean isWriteActionRequired() {
+    return false;
+  }
+
+  public void testConstVoidDeclaration() {
+    doTest(GoEmptyDeclarationInspection.QUICK_FIX_NAME, true);
+  }
+
+  public void testVarVoidDeclarationWithInnerComment() {
+    doTestNoFix("", true);
```

Comment:
`doTestNoFix` looks for quick fix named `""`. Looks like it's impossible to fail the test, use `GoEmptyDeclarationInspection.QUICK_FIX_NAME` here